python/pyspark/sql/tests/connect/streaming/test_parity_streaming.py

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+from pyspark.sql.tests.streaming.test_streaming import StreamingTestsMixin
+from pyspark.testing.connectutils import ReusedConnectTestCase
+
+
+class StreamingParityTests(StreamingTestsMixin, ReusedConnectTestCase):
```

Comment:
uhoh, the tests here shouldn't be skipped.

Reply:
This file marks all tests that should be skipped currently because these functionality is not supported yet in connect.

It executes all tests from `StreamingTestsMixin`, except these below marked skipped.

When anyone adds support to corresponding functions, they should delete the @unittest.skip and function, unless they want to do some change specific for connect.

So ideally in the end, this class should only be like 
```
class StreamingParityTests(StreamingTestsMixin, ReusedConnectTestCase):
    pass
```
And then it runs all tests from StreamingTestsMixin

Reply:
Each of the skipped tests here has a SPARK ticket associated with it. We will enable these soon. LGTM.